src/core/index.ts

```diff
@@ -130,14 +131,24 @@ export class DevAssist {
 			let assistantMessage = "";
 			try {
 				for await (const chunk of stream) {
+
 					assistantMessage += chunk.text;
 					this.assistantMessageContent = parseAssistantMessage(assistantMessage);
 					if (this.assistantMessageContent[this.assistantMessageContent.length - 1].type === "tool_use" && this.assistantMessageContent[this.assistantMessageContent.length - 1].name === "ask_followup_question") {
 						this.askFollowup = true;
 						this.askFollowupIndex = this.assistantMessageContent.length - 1;
 					}
-					// console.log("assistantMessageContent", this.assistantMessageContent);
+					if (this.assistantMessageContent[this.assistantMessageContent.length - 1].partial === true && this.assistantMessageContent[this.assistantMessageContent.length - 1].type === "text" && this.assistantMessageContent[this.assistantMessageContent.length - 1].content === '') {
+					// remove that element from the array becaus it is partial content
+					this.assistantMessageContent.pop();
+					}
+
```

Comment:
As we implemented this.currentStreamingContentIndex removing the empty text object is not necessary. I will update this code in next PR.